WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -21,15 +21,19 @@ @interface WPAnalyticsTrackerAutomatticTracks ()
 @property (nonatomic, strong) TracksService *tracksService;
 @property (nonatomic, strong) NSDictionary *userProperties;
 @property (nonatomic, strong) NSString *anonymousID;
+@property (nonatomic, strong) NSString *loggedInID;
 
 @end
 
 NSString *const TracksEventPropertyButtonKey = @"button";
 NSString *const TracksEventPropertyMenuItemKey = @"menu_item";
 NSString *const TracksUserDefaultsAnonymousUserIDKey = @"TracksAnonymousUserID";
+NSString *const TracksUserDefaultsLoggedInUserIDKey = @"TracksLoggedInUserID";
 
 @implementation WPAnalyticsTrackerAutomatticTracks
 
+NSString *_loggedInID, *_anonymousID;
```

Comment:
Can these be `@synthesize` statements instead of static variables?

```
@synthesize loggedInID = _loggedInID;
@synthesize anonymousID = _anonymousID;
```